src/sql/azdata.d.ts

```diff
@@ -4016,7 +4016,7 @@ declare module 'azdata' {
 		 * visualize: ResultSetSummary
 		 */
 		export interface QueryEventListener {
-			onQueryEvent(type: QueryEvent, document: queryeditor.QueryDocument, args: ResultSetSummary | string | undefined): void;
+			onQueryEvent(type: QueryEventType, document: queryeditor.QueryDocument, args: ResultSetSummary | string | undefined): void;
```

Comment:
Can you update the doc comment for `queryStop` event above to indicate it sends the queryInfo object? Other extenders may wish to use this

Reply:
Currently the queryInfo isn't being sent through all the way through the extensibility side of the API. This just gets the args (which are pulled from the IQueryEventObject). I'll need to add this in later once I move this to be in its own extension but for now I'm not going to modify the API surface unless we have a need to. 

---
In reply to: [311245256](https://github.com/microsoft/azuredatastudio/pull/6579#discussion_r311245256) [](ancestors = 311245256)